es-app/src/views/gamelist/VideoGameListView.cpp

```diff
@@ -240,37 +240,15 @@ void VideoGameListView::updateInfoPanel()
 		fadingOut = true;
 
 	}else{
-		std::string				video_path;
-		std::string				marquee_path;
-		std::string				thumbnail_path;
-		video_path 			= file->getVideoPath();
-		marquee_path 		= file->getMarqueePath();
-		thumbnail_path 		= file->getThumbnailPath();
-
-		if	(!video_path.empty() && (video_path[0] == '~'))
```

Comment:
I don't feel strongly about it either way, but removing the handling of '~' may break things for some users.  Everything else looks good to me.  I will try to test it out tonight.

Reply:
I removed that because that is already handled by resolvePath in createFromXML